itou/users/management/commands/sync_group_and_perms.py

```diff
@@ -140,7 +140,12 @@ def get_permissions_dict():
 
     return {
         "itou-admin": {**group_itou_admin_permissions},
-        "itou-admin-readonly": {**{model: PERMS_READ for model in group_itou_admin_permissions}},
+        "itou-admin-readonly": {
+            **{model: PERMS_READ for model in group_itou_admin_permissions},
+            **{
+                users_models.User: PERMS_READ | PERMS_EXPORT_CTA,
```

Comment:
C'est une solution efficace, mais elle me semble s'éloigner du choix d'inserer les permissions dans les dictionnaires `group_itou_admin_permissions` etc… puis de les exporter. 
Là, nous commençons à avoir des permissions à 2 endroits, dont une (`users_models.User: PERMS_READ`) duplique celle implicite dans `{model: PERMS_READ for model in group_itou_admin_permissions}`.

Reply:
Je ne vois pas vraiment de différence avec ce qui était fait : c'est bien dans ce dict que l'on construit les permissions, juste que précédemment on ignorait justement les permissions définies dans `group_itou_admin_permissions`, on ne récupérait que les modèles auxquels on forçait la seule permission `PERMS_READ`